Homework2/Trie/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace Trie
```

Comment:
```suggestion
﻿namespace Trie;
```
И дальше без фигурной скобки и отступа